cylc/flow/scripts/cylc.py

```diff
@@ -46,32 +46,24 @@ def get_version(long=False):
 
 
 USAGE = f"""{cylc_header()}
-Cylc ("silk") orchestrates complex cycling (and non-cycling) workflows.
+Cylc ("silk") efficiently manages distributed cycling workflows.
+This is Open Source software (GPL-3.0): see "cylc help license".
 
 Version:
-  $ cylc version --long           # print cylc-flow version and install path
+  $ cylc version --long
   {get_version(True)}
 
 Usage:
-  $ cylc help license             # view the Cylc license (GPL-3.0)
-  $ cylc help all                 # list all commands
-  $ cylc validate <workflow>      # validate a workflow configuration
-  $ cylc install <workflow>       # install a workflow
-  $ cylc play <workflow>          # run/resume a workflow
-  $ cylc scan                     # list all running workflows (by default)
-  $ cylc tui <workflow>           # view/control workflows in the terminal
-  $ cylc stop <workflow>          # stop a running workflow
-
-Command Abbreviation:
-  # Commands can be abbreviated as long as there is no ambiguity in
-  # the abbreviated command:
-  $ cylc trigger WORKFLOW//CYCLE/TASK    # trigger TASK in WORKFLOW
-  $ cylc trig WORKFLOW//CYCLE/TASK       # ditto
-  $ cylc tr WORKFLOW//CYCLE/TASK         # ditto
-  $ cylc t                               # Error: ambiguous command
+  $ cylc <sub-command> <OPTS> <ARGS>
```

Comment:
I think `cylc <sub-command> <OPTS> <ARGS>` is a bit too abstract to be useful.

The previous section was good for showing the life-cycle of a workflow but was too long. Perhaps consider a "Quick Usage" or "Quick Start" section.

```
Quick Start:
  $ cylc install <path>       # install a workflow
  $ cylc play <workflow_id>   # run/resume a workflow
  $ cylc stop <workflow_id>   # stop a workflow
  $ cylc clean <workflow_id>  # delete an installed workflow
  $ cylc help all             # view all Cylc commands
```

With a shorter list of commands and a more descriptive name the opportunity for confusion should be less (especially if we put `cylc help all` at the end).

Reply:
The problem with the original, is without careful reading it could (evidently!)  be taken as an exhaustive list of commands. 

I quite like "Quick Start:" though as it suggests "just the most important commands for basic usage".  I'll reformulate...

Reply:
Perhaps with that, we don' t need the "Selected Commands" list at the bottom, which most duplicated the list at top.

Reply:
Fair enough, the list is shorter now so the need to provide a short-list is less.